src/app/Main/Body/QueryPage/Results/TableResults/TableContent-RowWithColSums.component.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsx jsx */
+import { css, jsx, SerializedStyles } from "@emotion/core";
+import TableCell from "@material-ui/core/TableCell";
+import _ from "lodash";
+import {
+    backgroundColor,
+    bfrPrimaryPalette,
+    primaryColor,
+} from "../../../../../Shared/Style/Style-MainTheme.component";
+
+const tableCellStyle = (isName: boolean): SerializedStyles => css`
+    box-sizing: border-box;

```

Comment:
Consider using a separate .style file for shared styles between sub components. E.g. a ResultTable.styles file with common styles for the table. E.g. you could omit the magic number 160px then by defining this in the style file as e.g. `const headerColumnWidh = 160px`. Or define borders like: defaultBorderStyle and elevatedBorderStyle and so on. Idea is that it would become very easy to change those styles in one dedicated place.

Reply:
Done. Commit "Add shared style for results table"